package.json

```
@@ -149,6 +149,11 @@
                     "default": false,
                     "markdownDescription": "Whether to add a CodeLens to `BUILD`/`BUILD.bazel` files to provide actions while browsing the file."
                 },
+                "bazel.limitQuickPickQueriesToNearestPackage": {
+                    "type": "boolean",
+                    "default": false,
+                    "markdownDescription": "Limit quick pick queries to the nearest package."
```

Let's extend the description here a little bit, so that it is not the exact same as the heading. How about:
```suggestion
                    "markdownDescription": "If active, the selection dialog for targets or packages will limit the results to the current package."
```